crates/autopilot/src/arguments.rs

```diff
@@ -8,12 +8,23 @@ pub struct Arguments {
     #[clap(flatten)]
     pub shared: shared::arguments::Arguments,
 
+    #[clap(flatten)]
+    pub order_quoting: shared::arguments::OrderQuotingArguments,
+
     #[clap(flatten)]
     pub http_client: http_client::Arguments,
 
     #[clap(flatten)]
     pub token_owner_finder: token_owner_finder::Arguments,
 
+    /// Address of the ethflow contract
+    #[clap(long, env, default_value = "31172bb2b5f97e8e89cf3376495d7bc7252f5a53")]
+    pub ethflow_contract: H160,
```

Comment:
Does a default here make sense? I think we probably want a function that picks the default based on the network id and is called in autopilot's main function. Then we can still have an argument that allows overriding the address if you need this for testing.

Reply:
We want to deploy it determinstically. then it would have the same address on all networks. I think its okay